### DISCUSSION TITLE
Web Panel works, but the proxy doesn't

### DISCUSSION THREAD

**Discussion by RealHypnoticOcelot** [mitmproxy/mitmproxy#6838] [Q&A]
Hello,

So I'm trying to inspect the traffic of an app on my iOS device, so I'm trying to hook up mitmproxy to do it. Since my computer is an M1 Mac, I ran `pip install mitmproxy`(instead of brew) and then `mitmweb`. On my phone, I configured the Wi-Fi network to proxy through the IP address of my computer on port 8080. When I try to access [http://mitm.it/](http://mitm.it/) though, I just get the **If you can see this, traffic is not passing through mitmproxy** text. Is there something I'm doing wrong? Cloudflare WARP is disabled on both devices and no VPNs are running, if that makes a difference. I've also tried changing the port but that doesn't seem to be doing anything either. Am I missing a step?

**Comment by emanuele-em** [comment]
try also on your mac, does it work if you set the proxy on your mac (127.0.0.1:8080) and if you connect to mimt.it from your mac?
If it works correctly, the problem is likely related to the ip or to the network (make sure that both device are on the same local network)

**Reply by RealHypnoticOcelot** [reply]
Thanks, so it does connect on the Mac using the settings you mentioned. I'm not sure what the issue could be though on my phone, because the IP and network are correct from what I can tell? I've double checked the IP and can't tell what could be wrong with the way I'm doing it.

**Reply by emanuele-em** [reply]
The first thing to test in my opinion is to connect to mitm.it from the same device that runs mitmproxy to exclude mitmproxy from the list of possible problems

**Reply by RealHypnoticOcelot** [reply]
Yes, I tried connecting to mitmproxy on my mac which is also the device running mitmproxy. That worked, but my phone still doesn't. They're on the same network and the IP should be correct as far as I can tell. 

Going to `192.168.X.XX:XXXX` in the browser shows `502 Bad Gateway` on both devices, so I'm not sure if that's intended behavior or not but it does show that it's at least accessible on my phone but something's stopping it from actually working

**Reply by RealHypnoticOcelot** [reply]
So I pulled out my Ubuntu Server and put mitmproxy in a Docker container with the following compose:
```yaml
services:
  mitmproxy:
    image: mitmproxy/mitmproxy
    restart: unless-stopped
    ports:
      - '8080:8080'
      - '8081:8081'
    volumes:
      - "./mitmproxy:/home/mitmproxy/.mitmproxy"
    command: ["mitmweb", "--web-host", "0.0.0.0"]
    tty: true
```
Like before, I can access 192.168.X.XX:8081 from my browser, but 8080 gets stuck loading. Putting the port in the proxy settings has the same result accessing [http://mitm.it/](http://mitm.it/), where it says it's misconfigured. 

This applies to both instances, but when I try to access port 8080 from my browser, it floods the web panel with requests for that domain and its favicon. I'm not sure whether or not this is intended behavior either, but nor can I tell what I'm doing wrong.
